number_trivia_app/lib/features/number_trivia/presentation/pages/number_trivia_page.dart

```diff
@@ -0,0 +1,69 @@
+// ignore_for_file: unused_label
+
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+
+import '../../../../injection_container.dart';
+import '../bloc/number_trivia_bloc.dart';
+import '../bloc/number_trivia_state.dart';
+import '../widgets/widgets.dart';
+
+class NumberTriviaPage extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: Text(
+          'Number Trivia',
```

Comment:
Would it make sense to remove the comma from single non-named parameter calls?

Reply:
Seems this Text was not updated. Not sure it makes sense, given other Text() were changed to move string to same line